deployment/kubernetes/ktranslate-nr.yaml

```diff
@@ -104,23 +109,28 @@ kind: ConfigMap
 metadata:
   name: ktranslate-config
 data:
-  snmp.yml: |
+  snmp-base.yaml: |
     devices:
-      switch:
-        device_name: switch
-        device_ip: 10.10.0.10
-        flow_only: true
-        user_tags: {}
     trap:
       listen: 0.0.0.0:1620
-      community: hello
-      version: ""
-      transport: ""
+    discovery:
+      cidrs:
+      - 10.10.0.0/24
+      ignore_list: []
+      debug: false
+      ports:
+      - 161
+      default_communities:
+      - public
+      default_v3: null
+      add_devices: true
+      add_mibs: true
+      threads: 4
+      replace_devices: true
+      check_all_ips: true
     global:
```

Comment:
The [wiki](https://github.com/kentik/ktranslate/wiki/Sample-SNMP-config#global-config-section) indicates `global.retries` is a required field. Is that accurate?

Reply:
Maybe `mib_profile_dir` should be required in the wiki as well 🤔

Reply:
Good call out on retries. 0 retries is the default which seems non optimal -- that's why its required, but things will work with 0 retries.